docs/guides/load-api-data.mdx

```diff
@@ -2,294 +2,203 @@
+## Quickstart
+Want to dive in? Simply run the following command to clone a pre-built repo:
 
+<CodeGroup>
+```bash Mac/Linux
+git clone https://github.com/mage-ai/etl-demo mage-etl-demo \
+&& cd mage-etl-demo \
+&& cp dev.env .env && rm dev.env \
+&& docker compose up
+```
 
----
-
-## `1.` Create new pipeline
-
-1. Go to the pipelines list page (`/pipelines`).
-    This is the default page when navigating to Mage in your web browser.
-1. In the top left corner of the page, click the button labeled <b>+ New</b>,
-    then select the option labeled <b>Standard (batch)</b> to create a new pipeline.
-1. In the left vertical navigation, click the last link labeled <b>Pipeline settings</b>.
-1. Change the pipeline’s name to `ETL demo`.
-1. Click the button labeled <b>Save pipeline settings</b>.
-
-<Frame>
-    <img
-        alt="Create new pipeline"
-        src="https://github.com/mage-ai/assets/blob/main/tutorials/etl/create-new-pipeline.gif?raw=true"
-    />
-</Frame>
-
----
-
-## `2.` Load data from an API
-
-1. In the left vertical navigation, click the 1st link labeled <b>Edit pipeline</b>.
-1. Click the button labeled <b>+ Data loader</b>,
-    then hover over <b>Python</b>, and click the option labeled <b>API</b>.
-1. A dialog menu will appear. Change the block name to `load data`.
-1. Click the button labeled <b>Save and add block</b>.
-1. Paste the following code in the data loader block:
-
-    ```python
-    import io
-    import pandas as pd
-    import requests
-
-
-    @data_loader
-    def load_data_from_api(*args, **kwargs):
-        url = 'https://raw.githubusercontent.com/mage-ai/datasets/master/restaurant_user_transactions.csv'
-        response = requests.get(url)
-        return pd.read_csv(io.StringIO(response.text), sep=',')
-
-
-    @test
-    def test_row_count(df, *args) -> None:
-        assert len(df.index) >= 1000, 'The data does not have enough rows.'
-    ```
-
-1. Run the block by clicking the play icon button in the top right corner of the data loader block
-    or press 1 of the following keyboard shortcuts:
-
-        - `⌘ + Enter`
-        - `Control + Enter`
-        - `Shift + Enter`
+```bash Windows
+git clone https://github.com/mage-ai/etl-demo mage-etl-demo 
+cd mage-etl-demo
+cp dev.env .env
+rm dev.env
+docker compose up
+```
+</CodeGroup>
 
-1. After you run the block (⌘ + Enter), you’ll see a sample of the data that was loaded.
+Then navigate to `https://localhost:6789` in your browser to see the pipeline in action!
 
-    <Frame>
-        <img
-            alt="Load data"
-            src="https://github.com/mage-ai/assets/blob/main/tutorials/etl/load-data.gif?raw=true"
-        />
-    </Frame>
+## Tutorial
 
----
+<AccordionGroup>
+    <Accordion title="1️⃣ Create a pipeline">
+        Open the pipelines list page (`/pipelines`). This is the default page when navigating to Mage in your web browser. If you have Mage running, just click [here](https://localhost:6789/pipelines).
 
-## `3.` Visualize data
```

Comment:
why don't we keep the "Visualize data" section?

Reply:
I think it's tangential to the process of ETL. I haven't noticed a ton of users engaging with charts. 

I think we could make a separate guide that walks though charts, but for this demo my goal was to get users on board transforming data as fast as possible.

Reply:
ok, got it. we can make a separate guide for charts then.

Reply:
I'll make a to-do